src/Http/Cookie/Cookie.php

```diff
@@ -621,7 +621,7 @@ public function withNeverExpire()
     public function withExpired()
     {
         $new = clone $this;
-        $new->expiresAt = new DateTimeImmutable('1970-01-01 00:00:01');
+        $new->expiresAt = new DateTimeImmutable('1970-01-01 00:00:01', new DateTimeZone('UTC'));
```

Comment:
```suggestion
        $new->expiresAt = new DateTimeImmutable('@1');
```

Reply:
oops, sorry, don't show your changes :(
I've committed it.

Reply:
no problem